src/tracer.js

```diff
@@ -124,16 +124,20 @@ export const trace = ({
   switchOff,
   eventFilter,
 }) => userHandler => async (event, context, callback) => {
-  TracerGlobals.setHandlerInputs({ event, context });
-  TracerGlobals.setTracerInputs({
-    token,
-    debug,
-    edgeHost,
-    switchOff,
-    eventFilter,
-  });
-
-  startHooks();
+  try {
```

Comment:
safeExcute?

Reply:
This is the "main", here we also run the code of the user and we don't want to catch his exceptions.

Reply:
👍